spec/serializers/provider_serializer_spec.rb

```diff
@@ -84,4 +84,78 @@
 
     it { should eq provider.ucas_preferences.send_application_alerts_before_type_cast }
   end
+
+  describe 'contacts' do
+    describe 'admin contact' do
+      let(:provider) do
+        create :provider,
+               contacts: [admin]
+      end
+      let(:admin) { create(:contact, type: 'admin') }
+
+      subject { serialize(provider)['contacts'].find { |c| c[:type] == 'admin' } }
+
+      its([:name]) { should eq admin.name }
```

Comment:
whats with the square brackets?

`its(:name)` ?

Reply:
:name is returned by a hash rather than a method (:name) just returns an indifferent hash error.

Reply:
`its(:name)` -> `subject.name`
`its([:name])` -> `subject[:name]`